### ISSUE TITLE
Xenonids should have a default vision setting

### ISSUE THREAD

**Issue by QuietlyWhisper** [RMC-14/RMC-14#8970]
## Description

- Allows the player to set their default vision state (most useful for half vision) so it does not need to be adjusted every spawn.
https://github.com/RMC-14/RMC-14/pull/8150#discussion_r2443508883

**Comment by Thereisabear** [comment]
This has been implemented.
